catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/adapter/BrickAdapter.kt

```diff
@@ -0,0 +1,441 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2021 The Catrobat Team
```

Comment:
update copyright, will not be changed because it is a new file

Reply:
@Walcho1125 Thanks, done!